.github/workflows/main.yml

```diff
@@ -36,4 +36,4 @@ jobs:
 
       - name: Push images
         if: github.ref == 'refs/heads/master' && github.event_name != 'pull_request'
-        run: docker buildx bake --push
+        run: docker buildx bake --push --no-cache
```

Comment:
Why no cache? The `Build images` step doesn't use cache anyway, so `docker buildx bake --push` is efficient and always uses fresh cache.

Reply:
Leftover from testing, good catch.